tox.ini

```diff
@@ -73,9 +73,14 @@ deps =
     integration: pytest-operator==1.0.0b1
 commands =
     charm: mypy {[vars]src_path} {posargs}
-    lib: mypy --python-version 3.5 {[vars]lib_path} {posargs}
+    lib: mypy --python-version 3.8 {[vars]lib_path} {posargs}
+      /usr/bin/env sh -c 'for m in $(git diff --name-only {[vars]lib_path}); do \
```

Comment:
1. This command should have the `lib:` prefix so it only runs when running `tox -e static-lib` and not every testenv.
2. This check fails as soon as the modified lib file is staged (or committed). Which means CI won't ever fail. The easiest fix is to _add_ a diff to the main branch `git diff HEAD..main`.

Reply:
True, I'll fix this in the PRs I'm opening for the other repos and I'll open a fix for this one too!